osmaxx/excerptexport/models/output_file.py

```diff
@@ -50,5 +50,7 @@ def get_filename_display(self):
         return ''
 
     def get_absolute_url(self):
+        if self.file:
+            return self.file.url
         from django.core.urlresolvers import reverse
-        return reverse('excerptexport:download', kwargs=dict(uuid=self.public_identifier))
+        return reverse('excerptexport:export_detail', kwargs={'id': self.export.extraction_order.excerpt.id})
```

Comment:
I feel like the decision whether to use the raw download URL or the URL of a HTML view doesn't belong in this method and should instead be the concern of the caller(s).

Or maybe this function just needs to be renamed to account for this `get_absolute_url`-atypical behavior.